pages/index.tsx

```diff
@@ -21,6 +29,49 @@ export default function HomePage({ packageCount }: { packageCount?: number }) {
     setSearchInput('');
   }
 
+  async function autoComplete(query) {
+    try {
+        // fetch the data
+        const resPackages = await fetch(
+          `${API_URL}/search_packages?q=${query}&page=1&latest=1`,
+          {
+            headers: {
+              Accept: 'application/json',
+            },
+          },
+        );
+
+        const { packages } = await resPackages.json();
+        const relevantPackages = packages.filter((p)=>(p.score>2.5));
```

Comment:
> Left some comments for optimizations you can do! Functionality seems to work well when I try 👍 It's already a big improvement! 🔥
> 
> There is just one thing I noticed is that if I type 2 or 3 characters for example, it can show me 1 package suggestion, but if I add a new character, it starts to show me 5 suggestions. Shouldn't it already show 5 before?
> 
> Example: plot package, if I write `pl` or `plo`, I only get this one package <img alt="image" width="1004" src="https://user-images.githubusercontent.com/38893718/177871361-5656538f-c9bb-4d20-b935-e4f358fa09b3.png">
> 
> However if I write `plot`, I get 5 package suggestions: <img alt="image" width="1064" src="https://user-images.githubusercontent.com/38893718/177871409-8c3c9080-3a44-4c0d-8b83-f753a9d71ffa.png">

I think this is because I set the limits on the elasticsearch for packages the score defaults to 1.82... if the results are a low match. In order for the score to go above this number, the query has to be quite similar. (see picture below).
I can either:
- > Set the filter for the scores so that irrelevant results don't come up. This will mean that there are sometimes no suggestions
- > Don't filter the results so that there are always suggestions but they might be irrelevant
<img width="892" alt="Screenshot 2022-07-08 at 19 43 10" src="https://user-images.githubusercontent.com/107868502/178052191-3892a0fc-66f9-4e39-b4ec-4402316414f6.png">